src/databases/databases.js

```diff
@@ -2,58 +2,67 @@ var config = require('../config.js');
 var Sqlite3 = require('better-sqlite3');
 var fs = require('fs');
 var path = require('path');
+var Sqlite = require('./Sqlite.js')
+var Mysql = require('./Mysql.js')
 
 let options = {
   readonly: config.readOnly,
   fileMustExist: !config.createDatabaseIfNotExist
 };
 
-// Make dirs if required
-if (!fs.existsSync(path.join(config.db, "../"))) {
-  fs.mkdirSync(path.join(config.db, "../"));
-}
-if (!fs.existsSync(path.join(config.privateDB, "../"))) {
-  fs.mkdirSync(path.join(config.privateDB, "../"));
-}
-
-var db = new Sqlite3(config.db, options);
-var privateDB = new Sqlite3(config.privateDB, options);
-
-if (config.createDatabaseIfNotExist && !config.readOnly) {
-  if (fs.existsSync(config.dbSchema)) db.exec(fs.readFileSync(config.dbSchema).toString());
-  if (fs.existsSync(config.privateDBSchema)) privateDB.exec(fs.readFileSync(config.privateDBSchema).toString());
-}
-
-// Upgrade database if required
-if (!config.readOnly) {
-  ugradeDB(db, "sponsorTimes");
-  ugradeDB(privateDB, "private")
-}
-
-// Enable WAL mode checkpoint number
-if (!config.readOnly && config.mode === "production") {
-  db.exec("PRAGMA journal_mode=WAL;");
-  db.exec("PRAGMA wal_autocheckpoint=1;");
-}
-
-// Enable Memory-Mapped IO
-db.exec("pragma mmap_size= 500000000;");
-privateDB.exec("pragma mmap_size= 500000000;");
-
-module.exports = {
-  db: db,
-  privateDB: privateDB
-};
+if (config.mysql) {
+  module.exports = {
+    db: new Mysql(config.mysql),
+    privateDB: new Mysql(config.privateMysql)
+  };
```

Comment:
Should db upgrade be run on mysql?

Reply:
I could do it, however i was thinking that anybody using this would be basing it on the current live db version, so didn't want to. Can do if you think it's needed though.

so currently, i keep the service running but update the db in the background, so i'd expect manual intervention in the event of a schema change, but after rebuilding and deploying the service it'd work fine

Reply:
Oh okay, we can keep it out until if/when mysql is used on the main server

Reply:
That was my thinking yeah - didn't want to implement code i wouldn't be using for my use case - if you see merit in implementing it though, just say and i'll look into it.